src/Components/GraphVisualization.js

```diff
@@ -77,6 +77,8 @@ const GraphVisualization = (props) => {
       // })
```

Comment:
Remove commented part

Reply:
Some commented part is there for the labels on the links(lines) it is for future use so i have kept it as it is

Reply:
ok then Keep at it is.